_docs/tasks/telemetry/faq.md

```diff
@@ -9,3 +9,178 @@ type: markdown
 ---
 {% include home.html %}
```

Comment:
As discussed earlier:

- This should be part of the one-and-only FAQ. We can create different sections in this FAQ if needed. I don't think we want distributed FAQs within the site.

- Whether or not we keep this FAQ separate, we should follow the same model as the main FAQ. That is, one file per question which we then combine together via Jekyll voodoo. faq/index.md and _faq/* is how this can be done. I can help set this up if we want to keep distinct FAQ pages.

Reply:
@rshriram I believe you added the individual FAQ pages. Are you fine with the move to _faq ?

@geeknoid I was just trying to add content first and worry about any refactoring later. I'm fine moving this content to wherever now that it exists.